include/ringmesh/mesh/mesh.h

```diff
@@ -252,6 +248,7 @@ namespace RINGMesh
     protected:
         mutable std::unique_ptr< NNSearch< DIMENSION > > edge_nn_search_{};
         mutable std::unique_ptr< LineAABBTree< DIMENSION > > edge_aabb_{};
+        mutable AttributesManager edge_attributes_manager_;

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make this member variable "private". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3656)